lib/reports/cost_report.rb

```diff
@@ -145,14 +159,22 @@ def add_ht_item_to_freq_table(ht_item)
     end
 
     def matching_clusters
+      # We don't apply the icus rule here, since we're getting clusters not items.
+      # Any items gotten from these clusters need to be checked for icus though.
       if @organization.nil?
-        Cluster.where("ht_items.0": {"$exists": 1},
-          "ht_items.access": "deny").no_timeout
+        Cluster.where(
```

Comment:
appears to be only a formatting change rather than a logic change in `matching_clusters`, so presumably existing tests cover this?

Reply:
Covered, according to coveralls: https://coveralls.io/builds/62122116/source?filename=lib%2Freports%2Fcost_report.rb#L170

See also: https://github.com/hathitrust/holdings-backend/blob/782a666ab9c252005367e7cc09e0741055f1fbe4/spec/reports/cost_report_spec.rb#L126